pykafka/simpleconsumer.py

```diff
@@ -420,7 +420,7 @@ def __iter__(self):
                 return
             yield message
 
-    def consume(self, block=True):
+    def consume(self, block=True, rebalance_event=None):
```

Comment:
Using the name `rebalance` in the `SimpleConsumer` seems confusing to me, since the `SimpleConsumer` doesn't and shouldn't care that it's being used by a `BalancedConsumer`. A better name for this variable could be `unblock_event` - an event that, when set, causes `consume()` to return as soon as possible.